workbench/icons/Makefile.am

```diff
@@ -0,0 +1,20 @@
+icondir = $(datadir)/icons/hicolor/16x16/apps
```

Comment:
Yes, thanks. Works now. I added icons for 24x24, 32x32, 48x48 and scalable. The scalable icons are only symbolic which means they are painted in one color only (gray).

Now I got the problem the the icons for the first two levels (the project and the directories added to it) are much bigger than the icons for the folders and files contained below the directories. How can I control the size?